zerver/models.py

```diff
@@ -143,6 +143,9 @@ class Realm(ModelReprMixin, models.Model):
     mandatory_topics = models.BooleanField(default=False) # type: bool
     show_digest_email = models.BooleanField(default=True) # type: bool
     name_changes_disabled = models.BooleanField(default=False) # type: bool
+    allow_message_editing = models.BooleanField(default=True) # type: bool
+    DEFAULT_MESSAGE_EDIT_DURATION_SECONDS = 600
+    message_edit_duration_seconds = models.IntegerField(default=DEFAULT_MESSAGE_EDIT_DURATION_SECONDS) # type: int
```

Comment:
I would consider adding a newline before this block to set aside the "message editing related" section, but whatever works here :)